tests/fail/stacked_borrows/aliasing_mut1.stderr

```diff
@@ -1,8 +1,8 @@
-error: Undefined Behavior: not granting access to tag <TAG> because incompatible item is protected: [Unique for <TAG> (call ID)]
+error: Undefined Behavior: not granting access to tag <TAG> because incompatible item is protected: [Unique for <TAG>]
   --> $DIR/aliasing_mut1.rs:LL:CC
    |
 LL | pub fn safe(_x: &mut i32, _y: &mut i32) {}
-   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ not granting access to tag <TAG> because incompatible item is protected: [Unique for <TAG> (call ID)]
+   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ not granting access to tag <TAG> because incompatible item is protected: [Unique for <TAG>]
```

Comment:
Reporting the call ID is useful because you can use it with `-Zmiri-track-call-id`.
So if possible, it'd be nice if this info could be restored for the error messages.

Reply:
An attempt was made, though with slightly different formatting. I could hack in a way to get the old formatting in by formatting the `Item` manually, though I'm not sure how much people are attached to the old formatting.

Reply:
Unresolving since we had a thread somewhere to still tweak the format here (but I cannot find that other thread)

Reply:
I could have sworn I pushed a commit that made the change, but clearly I didn't.
https://github.com/rust-lang/miri/pull/2315#discussion_r915115275